src/middleware/companyAuth.middleware.ts

```diff
@@ -11,10 +11,10 @@ import SessionService from 'app/services/session/session.service'
 
 const OATH_SCOPE_PREFIX = 'https://api.companieshouse.gov.uk/company/'
 
-export default function CompanyAuthMiddleware(authConfig: AuthConfig,
-                                              encryptionService: JwtEncryptionService,
-                                              sessionService: SessionService,
-                                              logger: ApplicationLogger): RequestHandler {
+export default function CompanyAuthMiddleware(
+  authConfig: AuthConfig, encryptionService: JwtEncryptionService,
```

Comment:
I changed this because I was breaking the max-line-length lint rule.
I'll consistently make it this pattern across the controllers, though if I change this to server-level middleware, I shouldn't need to change this
```
@controller(
  URL,
  ...MIDDLEWARE1, MIDDLEWARE2
)
```